sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_loader_interface.cpp

```diff
@@ -0,0 +1,291 @@
+  pDdiTable->pfnMemImageWrite = nullptr;
+  pDdiTable->pfnMemImageCopy = nullptr;
+  pDdiTable->pfnMemBufferMap = nullptr;
+  pDdiTable->pfnMemUnmap = nullptr;
+  pDdiTable->pfnUSMMemcpy = nullptr;
+  pDdiTable->pfnUSMPrefetch = nullptr;
+  pDdiTable->pfnUSMMemAdvise = nullptr;
+  pDdiTable->pfnUSMFill2D = nullptr;
+  pDdiTable->pfnUSMMemcpy2D = nullptr;
+  pDdiTable->pfnDeviceGlobalVariableWrite = nullptr;
+  pDdiTable->pfnDeviceGlobalVariableRead = nullptr;
+
+  return retVal;
+}
+
+UR_DLLEXPORT ur_result_t UR_APICALL urGetEventProcAddrTable(
+    ur_api_version_t version, ///< [in] API version requested
+    ur_event_dditable_t
+        *pDdiTable ///< [in,out] pointer to table of DDI function pointers
+) {
+  auto retVal = validateProcInputs(version, pDdiTable);
+  if (UR_RESULT_SUCCESS != retVal) {
+    return retVal;
+  }
+  pDdiTable->pfnGetInfo = nullptr;
+  pDdiTable->pfnGetProfilingInfo = nullptr;
+  pDdiTable->pfnWait = nullptr;
+  pDdiTable->pfnRetain = nullptr;
+  pDdiTable->pfnRelease = nullptr;
+  pDdiTable->pfnGetNativeHandle = nullptr;
+  pDdiTable->pfnCreateWithNativeHandle = nullptr;
+  pDdiTable->pfnSetCallback = nullptr;
+
+  return retVal;
+}
+
+UR_DLLEXPORT ur_result_t UR_APICALL urGetKernelProcAddrTable(
+    ur_api_version_t version, ///< [in] API version requested
+    ur_kernel_dditable_t
+        *pDdiTable ///< [in,out] pointer to table of DDI function pointers
+) {
+  auto retVal = validateProcInputs(version, pDdiTable);
+  if (UR_RESULT_SUCCESS != retVal) {
+    return retVal;
+  }
+  pDdiTable->pfnCreate = nullptr;
+  pDdiTable->pfnGetInfo = nullptr;
+  pDdiTable->pfnGetGroupInfo = nullptr;
+  pDdiTable->pfnGetSubGroupInfo = nullptr;
+  pDdiTable->pfnRetain = nullptr;
+  pDdiTable->pfnRelease = nullptr;
+  pDdiTable->pfnGetNativeHandle = nullptr;
+  pDdiTable->pfnCreateWithNativeHandle = nullptr;
+  pDdiTable->pfnSetArgValue = nullptr;
+  pDdiTable->pfnSetArgLocal = nullptr;
+  pDdiTable->pfnSetArgPointer = nullptr;
+  pDdiTable->pfnSetExecInfo = nullptr;
+  pDdiTable->pfnSetArgSampler = nullptr;
+  pDdiTable->pfnSetArgMemObj = nullptr;
+  return retVal;
+}
+
+UR_DLLEXPORT ur_result_t UR_APICALL urGetMemProcAddrTable(
+    ur_api_version_t version, ///< [in] API version requested
+    ur_mem_dditable_t
+        *pDdiTable ///< [in,out] pointer to table of DDI function pointers
+) {
+  auto retVal = validateProcInputs(version, pDdiTable);
+  if (UR_RESULT_SUCCESS != retVal) {
+    return retVal;
+  }
+  pDdiTable->pfnImageCreate = nullptr;
+  pDdiTable->pfnBufferCreate = nullptr;
+  pDdiTable->pfnRetain = nullptr;
+  pDdiTable->pfnRelease = nullptr;
+  pDdiTable->pfnBufferPartition = nullptr;
+  pDdiTable->pfnGetNativeHandle = nullptr;
+  pDdiTable->pfnCreateWithNativeHandle = nullptr;
+  pDdiTable->pfnGetInfo = nullptr;
+  pDdiTable->pfnImageGetInfo = nullptr;
+
+  return retVal;
+}
+
+/// @brief
+/// @param version
+/// @param pDdiTable
+/// @return
+UR_DLLEXPORT ur_result_t UR_APICALL urGetPlatformProcAddrTable(
```

Comment:
```suggestion
UR_DLLEXPORT ur_result_t UR_APICALL urGetPlatformProcAddrTable(
```

Reply:
@bader  I addressed all your review comments but now CI is failing due to a new unrelated issue :(

I think this change in sycl-tests broke something https://github.com/intel/llvm-test-suite/commit/a92d98e5936defd9d16dd7c5bab4023c24915b38